app/src/main/java/com/kickstarter/viewmodels/ProfileViewModel.kt

```diff
@@ -80,13 +82,15 @@ interface ProfileViewModel {
         fun userNameTextViewText(): Observable<String>
     }
 
-    class ViewModel(environment: Environment) : ActivityViewModel<ProfileActivity>(environment), ProfileAdapter.Delegate, Inputs, Outputs {
-        private val client = requireNotNull(environment.apiClient())
-        private val currentUser = requireNotNull(environment.currentUser())
+    class ProfileViewModel(environment: Environment) : ViewModel(), ProfileAdapter.Delegate, Inputs, Outputs {
```

Comment:
same question as above, we should be clearing the disposables when the viewmodel is destroyed